packages/neuron-wallet/src/types/ckbComponents.d.ts

```diff
@@ -0,0 +1,280 @@
+import type { Agent as HttpsAgent } from 'node:https'
+import type { Agent as HttpAgent } from 'node:http'
+
+declare global {
+  declare namespace CKBComponents {
+    type DAO = string
+    type Hash = string
+    type Number = string
+    type Hash256 = string
+    type UInt32 = string
+    type UInt64 = string
+    type U256 = string
+    type Index = string
+    type Version = string
+    type Count = string
+    type Difficulty = string
+    type BlockNumber = string
+    type EpochInHeader = string
+    type Capacity = string
+    type ProposalShortId = string
+    type Timestamp = string
+    type Nonce = string
+    type Cycles = string
+    type Size = string
+    type OutputsValidator = 'default' | 'passthrough' | undefined
+    type RationalU256 = Record<'denom' | 'numer', string>
+    type ProposalWindow = Record<'closest' | 'farthest', BlockNumber>
+    type EpochNumberWithFraction = string
+    type EpochNumber = string
+    enum TransactionStatus {
+      Pending = 'pending',
+      Proposed = 'proposed',
+      Committed = 'committed',
+    }
+    type ScriptHashType = api.HashType
+    type DepType = 'code' | 'depGroup'
+    type JsonBytes = string
+    /**
+     * @typedef Bytes, keep consistent with CKB
+     * @description Bytes will be serialized to string
+     * @see https://github.com/nervosnetwork/ckb/blob/develop/util/jsonrpc-types/src/blockchain.rs#L19
+     */
+    type Bytes = string
+    type Since = string
+    interface Node {
+      url: string
+      httpAgent?: http.Agent
+      httpsAgent?: https.Agent
```

Comment:
Does it work here?

Reply:
I think it works fine, the typescript compiler is not complaining, test cases running fine, and I referred to previous code: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-wallet/src/utils/ckb-rpc.ts#L354C23-L354C23

Reply:
Does it mean the import is unnecessary here?

Reply:
I forgot to update the imported type alias, updated by [3decee3](https://github.com/nervosnetwork/neuron/pull/2843/commits/3decee385d22a7a7c40aafff71f2b1d03981b1e8)